config/prod.exs

```diff
@@ -18,6 +18,8 @@ config :code_corps, CodeCorps.Endpoint,
   force_ssl: [rewrite_on: [:x_forwarded_proto]],
   secret_key_base: System.get_env("SECRET_KEY_BASE")
 
+config :code_corps, System.get_env("SITE_URL")
```

Comment:
Why did we use this in this case but not the others? Would probably recommend doing codecorps.org here.